cluster-provision/images/vm-image-builder/README.md

```diff
@@ -5,18 +5,18 @@ would like to create KubeVirt VM's with customized container-disk,
 without the hassle of installing and customizing the OS manually.
 
 Use cases:
-- VM's for Demos
+- VMs for Demos
 - Pre-configured Kubevirt VM image for tests, for example:
     - Image with qemu-guest-agent installed and configured.
-    - Image with sriov drivers for sriov-lane images.
-    - Image with DPDK package for testing DPDK applications on KubeVirt.
+    - Image with SR-IOV drivers.
+    - Image with DPDK dependencies for testing DPDK applications on KubeVirt.
```

Comment:
Not related to the first commit / PR, if you can split it please to other commit ?
same for line 50 and 65